research/models.py

```diff
@@ -77,3 +105,35 @@ def __unicode__(self):
 
     def __str__(self):
         return self.title
+
+    @property
+    def bibtex(self):
+        if self.bibtex_string:
+            try:
+                return parse_string(self.bibtex_string, 'bibtex')
+            except:
+                pass
+
+        return None
+
+    @property
+    def citation(self) -> str:
```

Comment:
`citation()` has an expected return type of `str`, but can return `None`

Reply:
Removed type hinting. We don't use it anywhere and it just got added by because I've been using it elsewhere.